### ISSUE TITLE
Linux support

### ISSUE THREAD

**Issue by keilmillerjr** [AutomationDirect/ProductivityBlocks#11]
Any chance of of a Linux release? CI could compile a jar. An alternative could be compilation instructions. Thank you.

**Comment by yNosGR** [comment]
Yes please, a linux build would be greatly appreciated.Even if tech support isn't available,  I think most linux users would be OK with a "drop these jars in the '~/Arduino/tools/ProductivityBlocks/tool’ directory" type install.
